pkgs/applications/networking/mkchromecast/default.nix

```
@@ -0,0 +1,87 @@
+{ lib, fetchFromGitHub, python3Packages, makeWrapper, sox, flac, faac, lame
+, ffmpeg, vorbis-tools, nodejs, pulseaudio, alsaLib, alsaUtils, youtube-dl
+, opusTools, gstreamer
+, alsaSupport ? false
+, systemTray ? true
+, ffmpegSupport ? true
+, youtubeSupport ? true
+}:
+
+# We define non-python packages here so we can reuse the binding in
+# makeSearchPathOutput below.
+let packages = [
+      vorbis-tools
+      sox
+      flac
+      faac
+      lame
+      opusTools
+      gstreamer
+      nodejs
+    ] ++ lib.optional alsaSupport [ alsaLib ffmpeg ]
+      ++ lib.optional (!alsaSupport) pulseaudio
+      ++ lib.optional ffmpegSupport ffmpeg
+      ++ lib.optional youtubeSupport youtube-dl;
+
+in python3Packages.buildPythonApplication rec {
+  pname = "mkchromecast";
+  version = "0.3.8.1";
+
+  src = fetchFromGitHub rec {
+    owner = "muammar";
+    repo = pname;
+    rev = version;
+    url = "https://github.com/${owner}/${pname}/archive/${version}.tar.gz";
+    sha256 = "0baxilfalndip3fx8cj03pjkdywv1v65k41srarcdrzvm8dqca9s";
+  };
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  propagatedBuildInputs = let py = python3Packages; in [
+    py.PyChromecast
+    py.psutil
+    py.mutagen
+    py.flask
+    py.netifaces
+    py.requests
+  ] ++ lib.optional systemTray py.pyqt5 ++ packages;
+
+  # Build instructions are macOS specific in 0.3.8.1;
+  # master has Linux build instructions but awaiting new release.
+  dontBuild = true;
+
+  format = "other";
+
+  # Relies on an old version (0.7.7) of PyChromecast unavailable in Nixpkgs.
+  doCheck = false;
+
+  # Patch a relative path referring to a Javascript file, to be absolute.
+  patchPhase = ''
+    sed -i "s_\['\./nodejs_['$out/lib/nodejs_" mkchromecast/video.py
+  '';
+
+  installPhase = with lib; let
+    path = strings.makeSearchPathOutput "" "bin" (lists.remove nodejs packages);
```

```suggestion
    path = lib.strings.makeSearchPathOutput "" "bin" packages;
```
You can remove `nodejs` from `packages`.
Actually, I think as short as this is the `let` isn't needed.